src/Cache_Command.php

```diff
@@ -406,4 +409,168 @@ public function flush_group( $args, $assoc_args ) {
 		}
 		WP_CLI::success( "Cache group '$group' was flushed." );
 	}
+
+	/**
+	 * Get a nested value from the cache.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <key>
+	 * : Cache key.
+	 *
+	 * <key-path>...
+	 * : The name(s) of the keys within the value to locate the value to pluck.
+	 *
+	 * [--group=<group>]
+	 * : Method for grouping data within the cache which allows the same key to be used across groups.
+	 * ---
+	 * default: default
+	 * ---
+	 *
+	 * [--format=<format>]
+	 * : The output format of the value.
+	 * ---
+	 * default: plaintext
+	 * options:
+	 *   - plaintext
+	 *   - json
+	 *   - yaml
+	 * ---
+	 */
+	public function pluck( $args, $assoc_args ) {
+		list($key) = $args;
+		$group     = Utils\get_flag_value( $assoc_args, 'group' );
+
+		$value = wp_cache_get( $key, $group );
+
+		if ( false === $value ) {
+			WP_CLI::halt( 1 );
```

Comment:
It'd be good to have a functional test covering this scenario.

Reply:
This match the implementation of the option's  [`pluck` command ](https://github.com/wp-cli/entity-command/blob/main/src/Option_Command.php#L587-L589). Looking at it again here, I think it would be better to use a proper error message instead of just halting the command.
```suggestion
			WP_CLI::error( "Object with key '$key' and group '$group' not found." );
```

Reply:
Feature test for this case has been created.